lib/awskeyring_command.rb

```diff
@@ -417,7 +418,8 @@ def console(account = nil) # rubocop:disable Metrics/AbcSize, Metrics/MethodLeng
     if options['no-open']
       puts login_url
     else
-      pid = Process.spawn("open \"#{login_url}\"")
+      spawn_cmd = browser.nil? ? "open \"#{login_url}\"" : "open -a \"#{browser}\" \"#{login_url}\""
```

Comment:
Finally to use the method option use options['browser'].

Reply:
using options[:browser] now...

Still new to ruby @tristanmorgan, so thanks heaps for all ur feedback :)